CONTRIBUTING.md

```diff
@@ -22,3 +22,28 @@ Run `yarn lint --fix` to automatically format files.
 If you can, take look at [commands.md](commands.md) and check out [Wadler's paper](http://homepages.inf.ed.ac.uk/wadler/papers/prettier/prettier.pdf) to understand how Prettier works.
 
 If you want to know more about Prettier's GitHub labels, see the [Issue Labels](https://github.com/prettier/prettier/wiki/Issue-Labels) page on the Wiki.
+
+## Performance
+
+If you're contributing a performance improvement, the following Prettier CLI options can help:
+
+- `--debug-benchmark` uses [`benchmark`](https://npm.im/benchmark) module to produce statistically significant time measurements. The measurements are printed at the debug log level, use `--loglevel debug` to see them.
+- `--debug-repeat N` uses a naive loop to repeat the formatting `N` times and measures the average run time. It can be useful to highlight hot functions in the profiler. The measurements are printed at the debug log level, use `--loglevel debug` to see them.
+
+To run Prettier with the profiler, use the Node CLI `--inspect-brk` option and Chromium/Chrome browser's [`chrome://inspect`](chrome://inspect) page to activate the Chrome DevTools Profiler for Node:
+
+```
+$ cat FileToFormat.js | NODE_ENV=production node --inspect-brk ./bin/prettier.js --stdin-filepath FileToFormat.js --debug-repeat 1000 --loglevel debug > /dev/null
```

Comment:
Do you think it would make sense to have an npm script for this? Here’s my start:

```sh
cat $1 | NODE_ENV=production node --inspect-brk ./bin/prettier.js --stdin-filepath $1 --debug-repeat 1000 --loglevel debug > /dev/null
```

you could then do `yarn foo FileToFormat.js` and it would run the tests.

Reply:
Yes, I can add an npm script, but the user interaction will still be required to open the inspector and start profiling.

Reply:
Also, the 1000 probably deserves to be a CLI argument, and this opens a whole new can of worms because now there are two CLI arguments to parse.

Reply:
IMO even if user interaction is required, it’s still easier to run a script than to copy-and-paste that same long command each time. For the `--debug-repeat` value, you can do `${2:-1000}`.

Reply:
Agree.